baselines/audioworklet.asynciterable.generated.d.ts

```diff
@@ -0,0 +1,3 @@
+/////////////////////////////
+/// AudioWorklet Async Iterable APIs
+/////////////////////////////
```

Comment:
There aren't any async iterable APIs inside audio worklets yet. Do we want to emit an empty file, or remove this file instead?

Reply:
I'm okay with keeping this.